src/Runtime/OMTensorHelper.h

```diff
@@ -162,4 +211,12 @@ template <typename T>
 bool omTensorAreTwoOmtsClose(
     const OMTensor *a, const OMTensor *b, float rtol = 1e-5, float atol = 1e-5);
 
+/**
```

Comment:
Maybe I was confusing, I would suggest to move this one to the official interface as I think it will provide a convenient add-on to the services we currently have.  I did not care for some of the support ops, this one is potentially useful to users of the interface too.

You should add in the description if the printing goes to stdout or stderr.

Reply:
done